src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx

```diff
@@ -126,6 +126,14 @@ export function DAppsExplorerScreenSearchFilter() {
     ValoraAnalytics.track(DappExplorerEvents.dapp_screen_open)
   }, [])
 
+  const createOnPressDappHandler = (dapp: ActiveDapp, index: number) => () => {
+    onSelectDapp(dapp, {
+      position: 1 + index,
```

Comment:
i checked with Ellen if we should take into account the number of favorites when reporting the dapp's position https://linear.app/valora/issue/RET-822#comment-2ed0c83e for now, the answer is no because we care about the order to learn about our recommendations. the number / order of favorites that a user curates is not directly relevant here. would be nice to report that number too but that requires us to refactor the favourite search / filter results to be in this component rather than the Favorites component, but we also don't have a use case for that data yet so i haven't done it here